src/js/components/Banner/ShabadavaliBanner.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Banner from './Banner';
+
+const ShabadavaliBanner = () => {
+
+  const {
+    isFetchingData: isFetchingBannerMessage,
+    data: bannerData,
+  } = {
+    isFetchingData: false,
+    data: {
+      active: true,
+      data: {
+        message: "Help us preserve the Punjabi language! Donate on Dasvandh Network and double your impact with matching!",
+        link: "dvnetwork.org/learnpunjabi",
+        label: "Donate now",
+        type: "3",
+      }
+    }
+  };
+
+  const lastSeen = sessionStorage.getItem("bannerMessage");
```

Comment:
@MySmilingTurban Just wondering we are using the same localStorage item for multiple banners. Suppose a case where we wants to display another error banner due to some outage or so. In this case, if the localStorage item is already set to `seen`, we will miss this `outageBanner`. 

So suggestion will be to keep seperate state for banners. Maybe creating one object banners and having all the banner under that can help us.

Reply:
Hanji, makes sense
Thanks for pointing this out